dw-currency.js

```diff
@@ -82,6 +86,31 @@ export class DwCurrency {
     return `${beforeDecimal}${!afterDecimal ? '' : decimalSeparator + afterDecimal}`;
   }
 
+  /**
+   * Formats given currency value as string includes currency symbol.
+   * @param {Number | Object} value - Currency amount. If the first argument is the Object, then 
+   *    other input params are ignored and considered that input is specified as the Object.
+   * @param {String} currency - ISO code of the currency
+   * @param {String} position - Position of the symbol - posible values:  `prefix`, and `postfix`. default `prefix`.
+   * @returns 
+   */
+  static formatWithSymbol(value, currency, position) {
+    if(typeof value === 'object'){

```

Comment:
Code isn't properly formatted. Add a practice to auto-format the block of code you changed (not whole file).